src/day-1-toy/scope.py

```diff
@@ -20,11 +22,9 @@ def outer():
 
     def inner():
         y = 999
-
```

Comment:
Just like a the `global` keyword, there's another keyword that alerts Python to look for a variable which is _neither_ local nor global. Do you know what that might be?